flutter_appauth_platform_interface/lib/src/errors.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/services.dart';
+
+class FlutterAppAuthPlatformErrorDetails {
```

Comment:
Could API docs be added for the new classes and their properties?

Reply:
Thanks for making the changes. Continuing on this thread to leave a general comment. Could you look at https://dart.dev/effective-dart/documentation to see how the API docs could be updated to match. The ones that stand out to me so far around how some properties aren't  starting using a single sentence summary and could then be followed up with more information in separate paragraphs